src/main/java/picard/illumina/parser/TileMetricsUtil.java

```diff
@@ -111,13 +117,30 @@ public static File renderTileMetricsFileFromBasecallingDirectory(final File illu
                 final int phasingCode = IlluminaMetricsCode.getPhasingCode(descriptorIndex, IlluminaMetricsCode.PHASING_BASE);
                 final int prePhasingCode = IlluminaMetricsCode.getPhasingCode(descriptorIndex, IlluminaMetricsCode.PREPHASING_BASE);
 
+                float phasingValue = 0;
+                float prePhasingValue = 0;
+
                 if (!(codeMetricsMap.containsKey(phasingCode) && codeMetricsMap.containsKey(prePhasingCode))) {
-                    throw new PicardException("Don't have both phasing and prephasing values for tile");
+                    final String message = String.format(
+                            "Don't have both phasing and prephasing values for read %s cycle %s.  Phasing code was %d and prephasing code was %d.",
+                            isFirstRead ? "one" : "two", descriptorIndex+1, phasingCode, prePhasingCode
+                    );
+                    switch (validationStringency) {
+                        case LENIENT:
+                            LOG.warn(message);
```

Comment:
This happens to us as well, and no clue as to why.  We have been modifying the TileMetricsOut.bin to add 0s, so this is much better.  Thanks @nh13!